amazon-import.js

```diff
@@ -0,0 +1,305 @@
+// ==UserScript==
+// @name        BookBrainz: Import from Amazon
+// @include     *://www.amazon.*/*/dp/*
+// @version     0.0.1
+// @author      tr1ten
+// @description Import releases from Amazon
+// @run-at      document-end
+// ==/UserScript==
+
+"use strict";
+function GM_addStyle(css, id) {
+  const style =
+    document.getElementById(id) ||
+    (function () {
+      const style = document.createElement("style");
+      style.type = "text/css";
+      style.id = id;
+      document.head.appendChild(style);
+      return style;
+    })();
+  const sheet = style.sheet;
+  sheet.insertRule(css, (sheet.rules || sheet.cssRules || []).length);
+}
+// injecting css
+GM_addStyle(
+  `
+  .bb-btn{  
+    padding:0.4em;
+    margin:0.5em;
+      color: white;
+      background-color: #04aa6d;
+      border:none;
+
+  }
+`,
+  "bookbrainz"
+);
+GM_addStyle(
```

Comment:
You can combine all these stylesheet injections into a single call which injects all rules at once.